client/deposits/strings.ts

```diff
@@ -111,7 +111,7 @@ export const payoutFailureMessages: Record< PayoutFailureCode, string > = {
 		'woocommerce-payments'
 	),
 	issuer_unavailable: __(
```

Comment:
8. `issuer_unavailable` – Provide more accurate information about the automatic retry process and available options.